src/System.Private.CoreLib/src/System/Runtime/InteropServices/NonPortable.cs

```diff
@@ -168,6 +173,11 @@ public static bool IsComObject(object o)
             return false;
         }
 
+        public static bool IsTypeVisibleFromCom(Type t)
+        {
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_ComInterop);
```

Comment:
It may be more appropriate to return false here.

Reply:
Yes. just use similar logical as Marshal.IsComObject()